GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -33,9 +35,11 @@ public sealed class RevisionDataGridView : DataGridView
         internal RevisionGraph _revisionGraph = new RevisionGraph();
 
         private readonly List<ColumnProvider> _columnProviders = new List<ColumnProvider>();
-        private readonly AutoResetEvent _backgroundEvent = new AutoResetEvent(false);
-        private Thread _backgroundThread;
```

Comment:
😕 Why is this change still showing up as part of this pull request?

Reply:
This seems to be a GH issue. GE does not list this file in the comparison of this branch and master.